frappe/core/doctype/user/user.py

```diff
@@ -887,8 +886,7 @@ def get_active_users():
 
 def get_website_users():
 	"""Returns total no. of website users"""
-	return frappe.db.sql("""select count(*) from `tabUser`
-		where enabled = 1 and user_type = 'Website User'""")[0][0]
+	return frappe.db.count("User", filters={"enabled": "1", "user_type": "Website User"})
```

Comment:
Although seems only like a cosmetic change and doesn't seem to have any bearing if it's 1, True or "1", using `True` is still much cooler IMO 😭  

```suggestion
	return frappe.db.count("User", filters={"enabled": True, "user_type": "Website User"})
```